### ISSUE TITLE
Problem of generating Transaction

### ISSUE THREAD

**Issue by brockz** [tokenika/secure-bp-voting#9]
Hello 

i started generating my voting transaction without success.
When i click Generate Transaction on my air gapped pc ... nothing happens. No Signature is getting generated.
I tried IE, Edge and Chrome

Br Stefan

**Comment by noisy** [comment]
@brockz that means that form was not filled in a proper way.

1) For example you added some whitespace after/before your _account name_ / _private key_
2) Your private keys is incorrect (tip: private keys starts with `5....`, public keys starts with `EOS...`)
3) One of _chainId_ / _block_num_ / _ref_block_prefix_ is incorrect

**Comment by brockz** [comment]
hey noisy

i tried serveral times without success.
i used my account name (cross checked via eosauthority)
I used my private key starting with 5
i used these values:

![image](https://user-images.githubusercontent.com/25453305/41228805-e5762616-6d79-11e8-99f3-b67ce75307b1.png)

i tried as blockproducer:
tokenika4eos


no whitespaces

Br

**Comment by brockz** [comment]
I got your Problem.

In the 2_generate_transaction.html File you reference to:
<script src="https://github.com/EOSIO/eosjs/releases/download/v14.1.1/eos.js"></script>

This is not available on an airgapped pc !!!

I downloaded eos.js locally, verified the sha256 sum an changed your code to:
<script src=".\eos.js"></script> 

on the airgapped pc.
Working!

Br

**Comment by noisy** [comment]
> <script src="https://github.com/EOSIO/eosjs/releases/download/v14.1.1/eos.js"></script>
> This is not available on an airgapped pc !!!

Of course, that's why we provided an instruction:

![selection_592](https://user-images.githubusercontent.com/201263/41231945-11571a46-6d85-11e8-944d-83dc10517058.png)

Then, browser will download html file with eos.js file downloaded from linked location, as it is shown here at 6:06:

https://youtu.be/_sJYuXOUHCc?t=6m6s

----------------

I guess, your case is different, and you wanted to clone a repository and then use it, am I correct?

We decided to focus on instruction describing process which would not require to install/use git... or to download a whole repository in a zip file.. because that would require extra steps of downloading and unpacking.

But of course, your way of using it is still correct.
